### DISCUSSION TITLE
Improve the translation of English documentation

### DISCUSSION THREAD

**Discussion by lhapaipai** [lhapaipai/symfony-vite-dev#5] [General]
As many will have seen, the English version of the documentation contains numerous translation errors. If anyone would be motivated to lend a little helping hand that would be so great...
thanks in advance

**Comment by andyexeter** [comment]
Hi @lhapaipai,

I've just been through the docs and updated some of the translations and wordings.

I wanted to ask about this on the homepage:

> Give wings to your Sym[ph]on[ie] application.

What does the "Sym[ph]on[ie]" part mean? Is it a play on words with Symfony and Vite which I am not understanding? I propose we update this to "Symfony" unless there is something I am not getting :)

**Reply by lhapaipai** [reply]
oh it's a little joke...

Vite (French word for "quick") : https://vitejs.dev/guide/#overview
Vue (pronounced /vjuː/ like the french word) : https://vuejs.org/guide/introduction.html#what-is-vue

it seems that Evan You likes to use French terms for his package names.
as a result, many libraries followed the idea
TroisJS : https://troisjs.github.io/
etc...

with Symfony and its logo in the shape of a violin f-hole, the allusion was too tempting with the French term `Symphonie` (a musical composition : symphony in English).

in France developers pronounce `Symfony` in the same way as Symphonie is pronounced

afterwards if it only makes me laugh that is not the goal... what do you think

**Reply by lhapaipai** [reply]
![f-hole](https://github.com/lhapaipai/symfony-vite-dev/assets/1088155/91062af0-79ea-4a49-a88c-b3a38dfe41d6)

**Comment by andyexeter** [comment]
Also on the topic of translations, this strap line should say "developers" (or perhaps development?):

![Screenshot from 2024-03-11 21-29-56](https://github.com/lhapaipai/symfony-vite-dev/assets/6660584/86d514ed-64cd-459d-b7d3-f666741dd55d)

But I am unable to change it myself.

**Comment by lhapaipai** [comment]
Thank you so much @andyexeter !

**Reply by lhapaipai** [reply]
for my poor level of English I now find the translations absolutely perfect :smile:
